tests/testthat/test-renderer1-knit-print.R

```diff
@@ -1,164 +0,0 @@
-})
-
-## function to extract all circles from an HTML page
-get_circles <- function(html=getHTML()) {
-  plot.names <- c("plot1top", "plot1bottom")
-  count.vec <- c()
-  for(i in seq_along(plot.names)){
-    xpath <- sprintf("//div[@id='%s']//circle[@class='geom']", plot.names[[i]])
-    circle.list <- getNodeSet(html, xpath)
-    count.vec[[i]] <- length(circle.list)
-  }
-  count.vec
-}
-
-get_elements <- function(id){
-  list(a178=runtime_evaluate_helper(id=id, class_name='show_hide_selector_widgets', list_num=0),
-       b934=runtime_evaluate_helper(id=id, class_name='show_hide_selector_widgets', list_num=1),
-       show_hide=runtime_evaluate_helper(id=id, class_name='table.legend tr.label_variable', list_num=0),
-       widget=runtime_evaluate_helper(id=id, class_name='table.legend tr.label_variable', list_num=1))
-}
-
-plot1top <- get_elements("plot1top")
-plot1bottom <- get_elements("plot1bottom")
-
-# get_circles() returns a list of circles so comparing with list(expected, expected)
-
-test_that("clicking top legend adds/remove points", {
-  expect_equal(get_circles(), list(10, 10))
-  
-  clickID("plot1top_q_label_variable_a178")
-  expect_equal(get_circles(), list(5, 10))
-  
-  clickID("plot1top_q_label_variable_b934")
-  expect_equal(get_circles(), list(0, 10))
-  
-  clickID("plot1top_q_label_variable_b934")
-  expect_equal(get_circles(), list(5, 10))
-  
-  clickID("plot1top_q_label_variable_a178")
-  expect_equal(get_circles(), list(10, 10))
-})
-
-test_that("clicking bottom legend adds/remove points", {
-  expect_equal(get_circles(), list(10, 10))
-  
-  clickID("plot1bottom_q_label_variable_a178")
-  expect_equal(get_circles(), list(10, 5))
-  
-  clickID("plot1bottom_q_label_variable_b934")
-  expect_equal(get_circles(), list(10, 0))
-  
-  clickID("plot1bottom_q_label_variable_b934")
-  expect_equal(get_circles(), list(10, 5))
-  
-  clickID("plot1bottom_q_label_variable_a178")
-  expect_equal(get_circles(), list(10, 10))
-})
-clickSide <- function(position=NULL){
-  id <- paste0("plot1", position)
-  runtime_evaluate_helper(id=id, class_name='show_hide_selector_widgets', list_num=0, dispatch_event=TRUE)
-  runtime_evaluate_helper(id=id, class_name='selectize-input', list_num=0, dispatch_event=TRUE)
-}
-
-sendBackspace <- function() {
-  sendKey("Backspace")
-  Sys.sleep(0.5)
-}
-
-send <- function(alphabet) {
-  remDr$Input$insertText(text = alphabet)
-  sendKey("Enter")
-  Sys.sleep(0.5)
-}
-
-clickSide("top")
-test_that("top widget adds/remove points", {
-  expect_equal(get_circles(), list(10, 10))
-  sendBackspace()
-  expect_equal(get_circles(), list(5, 10))
-  sendBackspace()
-  expect_equal(get_circles(), list(0, 10))
-  send("a")
-  expect_equal(get_circles(), list(5, 10))
-  send("b")
-  expect_equal(get_circles(), list(10, 10))
-})
-
-clickSide("bottom")
-test_that("bottom widget adds/remove points", {
-  expect_equal(get_circles(), list(10, 10))
-  sendBackspace()
-  expect_equal(get_circles(), list(10, 5))
```

Comment:
this test file is important, why did you delete?

Reply:
knit-print test was not passing on GitHub actions before you merged the chromote testing PR (#126). but it passed on my local machine when I manually replaced phantom js  related files with chromote files. I will add it back now since the PR #126 is merged and my branch is updated with that changes.